exts/cesium.usd.plugins/schemas/cesium_schemas.usda

```diff
@@ -139,6 +139,38 @@ class CesiumSessionPrim "CesiumSessionPrim" (
     )
 }
 
+class CesiumIonServerPrim "CesiumIonServerPrim" (
+    doc = """Stores metadata related to Cesium ion server connections for tilesets."""
+    inherits = </Typed>
+    customData = {
+        string className = "IonServer"
+    }
+) {
+    string cesium:ionServerUrl (
+        customData = {
+            string apiName = "ionServerUrl"
+        }
+        displayName = "Cesium ion Server URL"
+        doc = "The hostname URL for the Cesium ion Server."
+    )
+
+    string cesium:projectDefaultIonAccessToken = "" (
+        customData = {
+            string apiName = "projectDefaultIonAccessToken"
+        }
+        displayName = "Project default ion Access Token"
+        doc = "A string representing the token for accessing Cesium ion assets."
+    )
+
+    string cesium:projectDefaultIonAccessTokenId = "" (
+        customData = {
+            string apiName = "projectDefaultIonAccessTokenId"
+        }
+        displayName = "Project default ion Access Token ID"
+        doc = "A string representing the token ID for accessing Cesium ion assets."
+    )
```

Comment:
Will `projectDefaultIonAccessToken` and `projectDefaultIonAccessTokenId` eventually be removed from `CesiumDataPrim`?

Reply:
Maybe. For now I think we should leave them here so people can retrieve the values they stored here.